Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -685,25 +686,23 @@ s32 WiiSockMan::DeleteSocket(s32 s)
 
 void WiiSockMan::Update()
 {
-  s32 nfds = 0;
```

Comment:
You shouldn't call `poll` if WiiSockets is empty. Otherwise, it spams error logs on Windows. Regardless, you still need to clear previous ufds values and update poll commands.